### ISSUE TITLE
Automatic score setting after a match

### ISSUE THREAD

**Issue by bartektartanus** [fred4jupiter/fredbet#47]
There is a free API that provides match results (up to 10 calls per minute). You just need to register to get a free API key and use it. The app could run every minute, check if any match is finished (more than 105 minutes after it started), and if so, call the API to get the results.

**Comment by bartektartanus** [comment]
Looks like this feature is not working - the match result is not updated. Logs:
```
2026-06-11 23:59:23,618 INFO  [de.fred4jupiter.fredbet.integration.FootballDataSyncService] - *** start syncing football data for competition Competition[id=2000, name=FIFA World Cup, code=WC, seasonYear=2026, type=CUP]
2026-06-11 23:59:23,903 DEBUG [de.fred4jupiter.fredbet.integration.FdMatchSyncImporter] - match with id=537327 is already up to date. No update needed. lastUpdate fdMatch=2026-06-11T21:09:14Z, lastUpdate match=2026-06-11T21:09:14Z
```

**Comment by fred4jupiter** [comment]
It depends on the result in the API call. I assume you use a free plan. In this plan the results are delayed. Please consult the docs at football-data.org

**Comment by bartektartanus** [comment]
Yes, indeed - I knew that results are delayed, but I've expected a little bit less delay :D Everything works fine, sorry for false bug report! Enjoy the World Cup!

**Comment by fred4jupiter** [comment]
Thank you! You too :-)

As the football-data.org API is integrated now I will close this issue.
